rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -27,6 +27,7 @@
 from cloudify.zip_utils import make_zip64_archive
 
 from manager_rest import utils, manager_exceptions, workflow_executor
+from manager_rest.constants import RESERVED_LABELS
```

Comment:
can't do all of these. Only `csys-consumer-id` is managed by the manager itself.